### ISSUE TITLE
Code blocks potentially fail 1.4.10 Reflow

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#166]
Where there are code blocks, for example on the [Language of Parts](https://alphagov.github.io/wcag-primer/sc/3.1.2.html) page, the code requires horizontal scrolling.

This seems to fail [1.4.10 Reflow](https://www.w3.org/WAI/WCAG22/Understanding/reflow), although there are counter-arguments:

- Default behaviour for code blocks is not to wrap text, so changing that could cause more problems.
- It could be argued that code should not wrap, to make it easier to understand. Reflow includes an exception in this case.

Here's how it looks on a 320 x 240 screen:

<img width="328" height="266" alt="HTML code block with text overflowing off to the right hand side" src="https://github.com/user-attachments/assets/1ad0a202-7599-4b7a-beb8-889161477742" />

To address the wrapping, I tried adding this to `/source/stylesheets/screen.css.scss` but it didn't do anything. However I'm not sure if this should be addressed as a WCAG issue or not. Any comments/guidance appreciated.

```CSS
// Style pre elements to wrap
.technical-documentation > pre {
  text-wrap-mode: wrap;
}
```

**Comment by andrewhick** [comment]
Comment from another accessibility specialist and developer in GDS: "Code that reflows is less usable and therefore fits into the exemption that it "require[s] two-dimensional layout for usage or meaning".
